RInAppMessaging/Classes/Extensions/UIApplication+IAM.swift

```diff
@@ -21,7 +21,15 @@ extension UIApplication {
                 .filter({ $0.activationState == .foregroundActive })
                 .compactMap({ $0 as? UIWindowScene })
                 .first {
-            return keyScene.statusBarManager?.statusBarStyle
+
+            var style = keyScene.statusBarManager?.statusBarStyle
+            // `default` style doesn't tell us anything, so we can fallback to the most common pattern:
+            // light status bar content in dark mode and dark content in light mode.
+            if style == .default, let keyWindow = getKeyWindow() {
+                let isDarkMode = keyWindow.traitCollection.userInterfaceStyle == .dark
```

Comment:
It seems like `userInterfaceStyle` is only available >= iOS 12?
https://developer.apple.com/documentation/uikit/uitraitcollection/1651063-userinterfacestyle

Reply:
True, but `.darkContent` doesn't. I would have to use my own enum to support that piece of logic on iOS 12. Do you think it's worth it?

Reply:
From the diff context I couldn't see that this was already wrapped in an `#available(iOS 13.0, *)` check. I was worried about a runtime crash. I tried putting the `let isDarkMode` line into an app and it wouldn't even compile without an availability check - Swift FTW! 😅